EazyShop/ESNav.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ESNav.swift
+//  EazyShop
+//
+//  Created by Cesar Rojas on 7/31/24.
+//
+
+import SwiftUI
+
+struct ESNav: View {
+    @State private var navigateToSignUp: Bool = false
+    
+    var body: some View {
+        NavigationView {
```

Comment:
Use NavigationStack instead NavigationView, as NavigationView is going to be deprecated as soon iOS 18 comes

Reply:
NavigationStack has been set!